custom_components/opensprinkler/config_flow.py

```diff
@@ -27,6 +36,10 @@
         vol.Required(CONF_PASSWORD): str,
         vol.Required(CONF_MAC): str,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
+        vol.Optional(CONF_MQTT_HOST): str,
```

Comment:
MQTT options here but it would probably work better in `OptionsFlow` and can be turned on and off?